sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -387,7 +387,7 @@ export class GeographyPoint {
     constructor(latitude: number, longitude: number);
     latitude: number;
     longitude: number;
-    toJSON(): object;
+    toJSON(): Record<string, unknown>;
```

Comment:
@xirzec I specifically want to high light this change. TS Linter objects to usage of object. (I could have disabled it for this line) ALternatively, this code change of changing it to ```Record``` solves the linter issue. And I don't think this changes any meaning of the code. I want to ask for your thoughts on this code change. Do you think this code change is ok?

Reply:
This looks fine to me.